### ISSUE TITLE
README - Installation (OSX) fixes

### ISSUE THREAD

**Issue by andriykorchak** [ericmjonas/pychirpz#9]
Hi @ericmjonas,

Thanks for this great library.

I tried installing FFTW and Boost using brew (as described in README), but `setup.py` is failing for me - it seems like it can't find their header files.

However, when I installed them using conda (`conda install -c conda-forge fftw boost`), I was able to successfully build and install pychirpz.

Do you mind if I change `Installation` section of the README - recommend using conda instead of brew (or add it as an alternative way of installation)? I can create a pull-request if you're ok with it.

I would also mention that it's recommended to install Cython before trying to install eigency (I couldn't get it working without this step).

Please let me know what you think about this. I'd be happy to contribute.

Thanks.

Andrii

**Comment by ericmjonas** [comment]
Yes, I'd totally take a PR! I'm sorry this wasn't working out of the box -- I still want to add a macos build to the circleci CI build but haven't gotten around to it yet.
